kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/checks/types/CheckContext.kt

```diff
@@ -153,11 +153,33 @@ public open class CheckContext<out T : Event>(public val event: T, public val lo
     /** Quick access to translate strings using this check context's [locale]. **/
     public fun translate(
         key: String,
-        bundle: String? = defaultBundle,
```

Comment:
It seems this behaviour has not been preserved?

Reply:
You're right, I have reverted this.